### DISCUSSION TITLE
GHChangelog Twitter Account discontinued?

### DISCUSSION THREAD

**Discussion by Andre601** [community/community#58705] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

I've noticed that the GHChangelog account on Twitter no longer is posting any updates about changes towards GitHub, which is sad as it's one of my main ways to get informed about updates that could be useful for me to know about.

I recall seeing IFTTT being defined as the device used for the tweets, so my guess is that IFTTT has discontinued their `X -> Twitter` system due to Twitter's decision of making the API paid?

Is there any chance for the account being back online in the future? Or can it be considered Discontinued at this point?
I've seen that the githubstatus accout is operating normally, so I hope the Changelog account will come back too or that there will be a proper alternative in the future (i.e. a Mastodon account? It seems to be a nice alternative to consider...)

**Comment by darkred** [comment]
I'm glad to see that it's back online since yesterday:
<details>
<summary>Screenshot:</summary>

![2023-08-03_195520](https://github.com/community/community/assets/723651/b3c25418-c339-4fac-94ea-8555ad9002b5)
</details>

The latest (today's) entry from the GitHub blog, ([link](https://github.blog/changelog/2023-08-03-request-a-cve-identifier-for-your-repository-security-advisory-via-rest-api)), is still missing, so I suppose the tweets are now posted manually.

**Reply by Andre601** [reply]
Probs.
Tweets sadly no longer display the tool/application used, so no way to see if it was automated or not.

I just wish they also are on other platforms like Mastodon...
